### ISSUE TITLE
Rel 3.4.2 - big changes in kitodo.config-properties

### ISSUE THREAD

**Issue by stefanCCS** [kitodo/kitodo-production#5078]
It looks like that with Rel 3.4.2 big changes in kitodo.config-properties were done.
A "diff" to previous version you can find here:
[kitodo-config.properties.diff.zip](https://github.com/kitodo/kitodo-production/files/8561980/kitodo-config.properties.diff.zip)
It is very hard to follow these changes and to "merge" it to the own configuration.
E.g. some lines were just moved ("directory.modules"), which makes any "diff" not usable ...
Also, it looks a bit, that might be a development kitodo.config-properties was checked-in for this Release instead of the "official" one (e.g. you can find a lot of Windows paths, which were linux paths before).
So,  please check, if this is "just" a check-in failure, and provide "official" kitodo-config-properties.
And/or please provide hints, which of these changes are really mandatory to take over in own config (resp. please explain a bit these changes).

**Comment by Kathrin-Huber** [comment]
Seems like I accidentially build the release with my lokal configurations, I'm going to fix the files in the release

**Comment by stefanCCS** [comment]
Ok , understood. @Kathrin-Huber , please feel free to close this issue, whenever you think this is done ...

**Comment by Kathrin-Huber** [comment]
The files are fixed 👍
